src/cpp/src/visual_language/vlm_config.cpp

```diff
@@ -19,6 +19,9 @@ ov::genai::VLMConfig::VLMConfig(const std::filesystem::path& json_path) {
 
     // Setting llava_next specific config params
     read_json_param(parsed, "image_newline", image_newline);
+    if (parsed.contains("vision_config")) {
+        read_json_param(parsed.at("vision_config"), "patch_size", vision_config_patch_size);
+    }
```

Comment:
`ProcessorConfig` already has `patch_size` field. Why doesn't `ProcessorConfig` check for `vision_config` in json?